integration/viewer/src/di.config.ts

```diff
@@ -47,11 +48,11 @@ export default function createContainer(options: IvyDiagramOptions): Container {
         deletionToolModule,
         edgeEditToolModule,
         nodeCreationToolModule,
-        ivyToolBarModule
+        ivyToolBarModule,
+        ivyAccessibilityModule
```

Comment:
not sure if this is important for the viewer, but not sure

Reply:
here it's getting removed for the viewer